examples/website-demos/social-messaging/src/components/Thread/ThreadHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useChannelActionContext, useChannelStateContext } from 'stream-chat-react';
+
+import { CloseX, Ellipse } from '../../assets';
+
+export const ThreadHeader: React.FC = () => {
+  const { thread } = useChannelStateContext();
+  const { closeThread } = useChannelActionContext();
+
+  if (!thread) return null;
+
+  return (
+    <div className='dm-header-container'>
```

Comment:
maybe we can update these css classes with more accurate names? DM doesn't really reflect what it is for the social app.